python2/typing.py

```diff
@@ -598,16 +606,6 @@ class Manager(Employee): pass
         Union[Manager, int, Employee] == Union[int, Employee]
         Union[Employee, Manager] == Employee
 
-    - Corollary: if Any is present it is the sole survivor, e.g.::
-
-        Union[int, Any] == Any
-
-    - Similar for object::
-
-        Union[int, object] == object
```

Comment:
I'd keep this even though it's not a special case -- it's a nice edge case that confirms the rule.